watchman/management/commands/watchman.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+import json
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+from watchman.utils import get_checks
+
+
+class Command(BaseCommand):
+    help = 'Runs the default django-watchman checks'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c',
+            '--checks',
+            dest='checks',
+            help='A comma-separated list of watchman checks to run (full python dotted paths)'
+        ),
+        make_option(
+            '-s',
+            '--skips',
+            dest='skips',
+            help='A comma-separated list of watchman checks to skip (full python dotted paths)'
+        ),
+    )
+
+    def handle(self, *args, **options):
+        check_list = None
+        skip_list = None
+        verbosity = options['verbosity']
+        print_all_checks = verbosity == '2' or verbosity == '3'
+
+        checks = options['checks']
+        skips = options['skips']
+
+        if checks is not None:
+            check_list = checks.split(',')
+
+        if skips is not None:
+            skip_list = skips.split(',')
+
+        for check in get_checks(check_list=check_list, skip_list=skip_list):
+            if callable(check):
+                resp = json.dumps(check())
```

Comment:
I actually did this as a bit of a shortcut.. the data structure I've got is different depending on the type of check.

eg. `databases` will be a dict of databases (`default`, `bookkeeping_shard_001`, etc.) each of which has it's own `'ok: True'` or `ok: False`, whereas email is a simple dict.

So I just figured that searching the serialized version for the failing string would be a simple way of doing it, rather than knowing how to parse my way through the trees.